sql/database.sql

```diff
@@ -3245,7 +3245,7 @@ CREATE TABLE `layout_options` (
   `form_id` varchar(31) NOT NULL default '',
   `field_id` varchar(31) NOT NULL default '',
   `group_id` varchar(31) NOT NULL default '',
-  `title` varchar(63) NOT NULL default '',
+  `title` text NOT NULL default '',
```

Comment:
```
`title` text,
```

Reply:
text defaults to null doesn't it? and init needs to be a string.